nautobot/users/migrations/0010_user_default_saved_views.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.14 on 2024-07-15 19:39
```

Comment:
Do you also need to delete `nautobot/users/migrations/0010_savedview.py` in this PR?

Reply:
GitHub is showing that one as a rename (to `/extras/migrations/0113_saved_views.py`) rather than a delete-and-add

Reply:
Ahh I didn't notice that. Thanks!